commerce_coordinator/apps/commercetools/pipeline.py

```diff
@@ -38,3 +39,30 @@ def run_filter(self, params, order_data):  # pylint: disable=arguments-differ
         return {
             "order_data": order_data
         }
+
+
+class UpdateCommercetoolsOrderReturnPaymentStatus(PipelineStep):
```

Comment:
Is there a separate ticket for utilizing this pipeline step?

Reply:
Yes, that will be done as part of a separate ticket to integrate all of the automatic refunds work together.